vendor/engines/sanger_sequencing/spec/presenters/sanger_sequencing/well_plate_presenter_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe SangerSequencing::WellPlatePresenter do
+  let(:samples) { FactoryGirl.build_stubbed_list(:sanger_sequencing_sample, 10) }
+
+  describe "sample_rows" do
+    let(:mapping) do
```

Comment:
If you pull this out into its own factory then you're able to use that same data structure in your frontend tests too. Might not be important enough to warrant the extraction, but thought I would put it out there.